service/dap/server_test.go

```diff
@@ -2502,6 +2502,10 @@ func TestBadLaunchRequests(t *testing.T) {
 		}
 
 		// Test for the DAP-specific detailed error message.
+		client.LaunchRequest("", fixture.Path, stopOnEntry)
```

Comment:
As the comment on previous line states, this section is for dap specific messages that come from the dap package. I moved tests for error messages that come from elsewhere to a separate section to emphasize that sometimes I can't test for specific error message because it's OS-specific and in general out of our control. Could you please move this down to around line 2560. Or you might not need this at all given #2365.

Reply:
Just wanted to add that it is not obvious what's wrong with fixutre.Path from just reading the code (i.e. that it will result in `can't load package: package /var/folders/dn/8hszkj9x21n__1gr_p8qgjgr0007jz/T/increment.f7f7ea9f: cannot find package`), which is why some of the tests below use something a bit more obvious that also results in a build error.

Reply:
I moved the comment as requested to line 2560

Reply:
This was a misunderstanding. The comment needs to stay where it was. It was the test that I suggested moving. If my other PR goes in, we won't need it at all, btw.